<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detailpage</class>
 <widget class="QMainWindow" name="Detailpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>1100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>1100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>详情页</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>601</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_number">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>铃声</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_music">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_address">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>地址</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_company">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>公司</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>531</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>邮箱</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>521</width>
       <height>47</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_birth">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>生日</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>521</width>
       <height>47</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit_social">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>社交</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/white.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>备注</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>950</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>收藏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>分享联系人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>保存修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-11</y>
      <width>1051</width>
      <height>1291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bg5.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>pushButton</zorder>
   <zorder>widget_7</zorder>
   <zorder>widget_6</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
